### ISSUE TITLE
Add Lewis Library application

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/lockers_and_study_spaces#109]
### User story

As a student, I would like to fill out an application for a locker at the Lewis Library with questions specific to the Lewis Library.

### Current application form

The current application form is at https://library.princeton.edu/lewis/services/locker-request

### Implementation notes, if any

The Lewis Library form should have these fields that are the same as the Firestone one:

- Semester of Occupancy
- Staff/Faculty/Student status
- Department
- Net ID
- Accessible (Clarify in issue #125)

The Preferred General Area should have two options:
- 3rd floor
- 4th floor

Fields from the Firestone Form that are not needed for Lewis:

- Preferred Size

Put this behind the existing patron feature flipper.  See issue #111.

**Comment by kevinreiss** [comment]
Display the size of the lewis locker on the form.

**Comment by kevinreiss** [comment]
We may have to update the existing floor label values as part of this implementation.

**Comment by maxkadel** [comment]
I started looking at this, but I think #108 needs to be done first (I'm not sure how the user will be routed to each application yet)

**Comment by maxkadel** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Feature-Work-Cycle-Board-571691cab409d8d821b873be/issues/pulibrary/lockers_and_study_spaces/109?planning-poker) with Zenhub @christinach @kevinreiss @regineheberlein @rladdusaw @sandbergja

**Comment by maxkadel** [comment]
Store size in database as 2 (foot), but display as "25 x 12"
